src/icons/file-icon/FileIcon.js.flow

```diff
@@ -142,7 +142,21 @@ const EXTENSIONS = {
     IconFileKeynote: mirror(['key']),
     IconFileNumbers: mirror(['numbers']),
     IconFilePages: mirror(['pages']),
-    IconFileImage: mirror(['bmp', 'gif', 'gdraw', 'jpeg', 'jpg', 'png', 'ps', 'svs', 'svg', 'tif', 'tiff']),
+    IconFileImage: mirror([
+        'bmp',
+        'gif',
+        'gdraw',
+        'jpeg',
+        'jpg',
+        'png',
+        'ps',
+        'svs',
+        'svg',
+        'tif',
+        'tiff',
+        'heic',
+        'heif',
+    ]),
```

Comment:
Oh yeah I'm not sure what the protocol is for updating files with both .flow and .tsx versions. Do I just copy the change to both?

Reply:
As far as I can tell from other flow/tsx files in BUIE the change goes to both. @booc0mtaco can confirm

Reply:
yeah, we want to mirror the changes in both for the time being.

- new components do not need to have a .flow file. typescript-only for new content
- existing components should keep the two in sync (esp. regarding type info... that's why they are still useful, i think.)